src/es5processor.ts

```diff
@@ -186,10 +187,18 @@ class ES5Processor extends Rewriter {
         }
         // Check for
         //   Object.defineProperty(exports, "__esModule", ...);
-        if (this.isEsModuleProperty(node as ts.ExpressionStatement)) {
+        if (this.isEsModuleProperty(exprStatement)) {
           this.emitCommentWithoutStatementBody(node);
           return;
         }
+        // Check for
+        //    module.exports = ...;
```

Comment:
super nit: use a 2-space tab here and below

Reply:
Done.